cmd/provisioner-localpv/app/helper.go

```diff
@@ -81,13 +87,90 @@ func (p *Provisioner) getPathAndNodeForPV(pv *v1.PersistentVolume) (path, node s
 	return path, node, nil
 }
 
+// createInitPod launches a helper(busybox) pod, to create the host path.
+//  The local pv expect the hostpath to be already present before mounting
+//  into pod. Validate that the local pv host path is not created under root.
+func (p *Provisioner) createInitPod(pOpts *HelperPodOptions) (err error) {
```

Comment:
Is this specific to hostpath? or is it applicable to device paths as well?

Reply:
currently it holds for hostpath, but might be needed for device paths as well.